### DISCUSSION TITLE
Kicking off community calls

### DISCUSSION THREAD

**Discussion by denelon** [microsoft/winget-cli#3765] [Community Call]
Hey everyone,

I've been considering the idea of a community call. 

Several of the product teams focusing on developers have been working together to engage directly with you (our customers). During the last [WinUI Community Call](https://www.youtube.com/watch?v=_m-EnEfG88g) we made an announcement ([about 22 minutes into the call](https://www.youtube.com/live/_m-EnEfG88g?si=U5ukUbScpQiPratE)) to bring more of the product teams together to share news and offer Q & A time.

We're thinking a bit about the cadence and what topics we would cover.

This is still work in progress, but I wanted to get some feedback to see if this is something you would be interested in.

**Comment by mdanish-kh** [comment]
Are you envisioning this as a big community call for all of the Windows developer teams or separate calls for each of the products (winget, powertoys, terminal...etc)? Regardless I'm all for it!

**Reply by denelon** [reply]
I'm expecting quarterly calls with the larger development team giving multiple updates for multiple products. I'm expecting a monthly cadence, and the team(s) on that call will largely be driven by community request. I would likely have updates every month since we do preview releases and experimental features we would want to share about and get feedback on.

**Comment by dotMorten** [comment]
The WinUI community calls were always a bit weird to me, because WinUI isn’t run as a community project and the community projects and blogs were never really promoted either, nor very much included in the actual call. I think there’s a lot you can learn from how for instance the .NET MAUI team runs their community calls

**Comment by riverar** [comment]
Don't think there's enough winget velocity or a large enough community to justify a community call. I just peek in here from time to time when I need to update a manifest.

**Comment by jantari** [comment]
I'd certainly be interested in such a call, both for hearing some progress and priority updates but - fair warning - also for asking about these: https://github.com/microsoft/winget-cli-restsource/issues?q=is%3Aissue+author%3Ajantari and Windows Server progress

**Comment by Karl-WE** [comment]
any news on this idea? Maybe starting with MVP DL first? Which one to choose? Now with the WS support I think this can get traction for different experts like Intune, AVD, WS

**Reply by denelon** [reply]
@Karl-WE I've got a Teams chat running with a few MVPs. Send me an e-mail and I'll see if I can add you to that discussion. We had some internal re-organization so the shared call with the other product teams is being re-evaluated.

**Reply by davidldennison** [reply]
If this is still in the works, would love to join!

**Comment by denelon** [comment]
@davidldennison, we've been having some "coffee chat" and other types of meetings. I'll send an e-mail so we can try to get you on the list.

**Reply by davidldennison** [reply]
That would be awesome! Email is [`david@daviddennison.com`](mailto:david@daviddennison.com)

Thank you! 🚀

**Comment by Gijsreyn** [comment]
I would love to see community calls around the winget area. Can you add me to the list of what's currently available @denelon?
